src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1550,7 +1550,7 @@ private static void CleanupDatabaseRelatedValues(ClusterOperationContext context
                 context.Transaction.InnerTransaction.OpenTable(IdentitiesSchema, Identities).DeleteByPrimaryKeyPrefix(databaseSlice);
             }
 
-            databaseLowered = $"{databaseName.ToLowerInvariant()}{SpecialChars.RecordSeparator}";
+            databaseLowered = $"{databaseName.ToLowerInvariant()}{(char)SpecialChars.RecordSeparator}";
             using (Slice.From(context.Allocator, databaseLowered, out var databaseSlice))
```

Comment:
IIRC, there is a .From method where you can pass the separator?

Reply:
Exactly. Thank you for the hint.
Done.